db/update.sql

```diff
@@ -27,3 +28,15 @@ ALTER TABLE `User`
   ADD `organization_web` varchar(255) DEFAULT NULL;
 
 -- -----------------------------------------------------------------------------  
+-- Events information table
+CREATE TABLE IF NOT EXISTS `Events` (
+  `event_id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `description` text NOT NULL,
+  `organizer` varchar(255) NOT NULL,
+  `event_date` int(11) NOT NULL,
+  `event_time` int(11) NOT NULL,
```

Comment:
@hongquan : `event_date` is for date on which the event is and `event_time' is the starting time of the event. 
In all the other tables date and time are stored in integer type only. I was trying to follow that convention only.
`DROP TABLE IF EXISTS `Shifts`;
CREATE TABLE IF NOT EXISTS `Shifts` (
  `SID` int(11) NOT NULL AUTO_INCREMENT,
  `title` text,
  `shifttype_id` int(11) NOT NULL,
  `start` int(11) NOT NULL,
  `end` int(11) NOT NULL,
  `RID` int(11) NOT NULL DEFAULT '0',
  `URL` text,
  `PSID` int(11) DEFAULT NULL,
  `created_by_user_id` int(11) DEFAULT NULL,
  `created_at_timestamp` int(11) NOT NULL,
  `edited_by_user_id` int(11) DEFAULT NULL,
  `edited_at_timestamp` int(11) NOT NULL,
  PRIMARY KEY (`SID`),
  UNIQUE KEY `PSID` (`PSID`),
  KEY `RID` (`RID`),
  KEY `shifttype_id` (`shifttype_id`),
  KEY `created_by_user_id` (`created_by_user_id`),
  KEY `edited_by_user_id` (`edited_by_user_id`)
) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=133 ;
`

Should I change it to `DATETIME` type? or follow the existing pattern?